Lib/fontParts/test/test_glyph.py

```diff
@@ -395,9 +395,19 @@ def test_get_contour_invalid(self):
     def test_appendContour_offset_valid(self):
         glyph = self.getGlyph_generic()
         contour = self.get_generic_object("contour")
-        glyph.appendContour(contour, (50, 50))
+        contour.insertPoint(0,position=(0,0))
+        contour.insertPoint(1,position=(100,100))
+        contour.insertPoint(2,position=(0,100))
+
+        self.assertEqual(len(glyph), 2)
+        glyph.appendContour(contour, (45, 50))
         self.assertEqual(len(glyph), 3)
 
+        newcontour = glyph[2]
```

Comment:
appenContour returns the latest added contour, use that instead of asking the glyph for a fixed contour

`newcontour  = glyph.appendContour(contour, (45, 50))`

Reply:
Asking the glyph checks that it actually made it into the glyph.

Reply:
True, so check assert both :)